libs/sdk-backend-tiger/src/convertors/fromBackend/afm/dimensions.ts

```diff
@@ -76,7 +75,7 @@ function transformDimension(
                             const newItem: IMeasureDescriptor = {
                                 measureHeaderItem: {
                                     localIdentifier: m.measureHeaderItem.localIdentifier,
-                                    name: m.measureHeaderItem.name ?? m.measureHeaderItem.localIdentifier,
+                                    name: m.measureHeaderItem.localIdentifier,
```

Comment:
Is the removal of the `??` part safe?

Reply:
api doesn't send `m.measureHeaderItem.name`, but `m.measureHeaderItem.localIdentifier` is required in api response....

Reply:
ok